types/kotlin2/Screen.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.analytics.plan
+
+import im.vector.app.features.analytics.itf.VectorAnalyticsScreen
+
+// GENERATED FILE, DO NOT EDIT
+
+/**
+ * Triggered when the user changed screen
+ */
+data class Screen(
+    val durationMs: Double? = null,
```

Comment:
This event is generally triggered when the user enter to the screen, so we do not have the duration yet. This parameter will probably not be used by the Android app.